dev-tools/check_file_permissions.sh

```diff
@@ -5,7 +5,7 @@ files_with_wrong_permissions=$(
     git ls-files --stage . \
         ':!*.sh' \
         ':!php-cs-fixer' \
-    | grep -P "100\d\d(1|3|5|7) " \
+    | grep -P "^100\d\d(1|3|5|7) " \
```

Comment:
that was a false-positive:
![Screenshot from 2020-10-27 00-12-30](https://user-images.githubusercontent.com/2716794/97238574-fa5ab700-17e9-11eb-9d74-4be60e5f679b.png)